test-trusted-directory/kustomize/overlay/dev-aic/kustomization.yaml

```diff
@@ -1,4 +1,4 @@
-namespace: devops-core
+namespace: dev-aic-core
```

Comment:
git compare confused again?

Reply:
No this one was wrong, it shouldnt have been devops but dev - but I don't think this branch is in use for the now dev-cdk environment, it uses master branch, which is why i've only now spotted this

(Master branch is used for envs that run v3, and ob-v4 branch is used when v4 is required)